packages/adapters-library/src/scripts/rpcInterceptor.ts

```diff
@@ -0,0 +1,119 @@
+import { createHash } from 'node:crypto'
+import { http, bypass } from 'msw'
+import { setupServer } from 'msw/node'
+import { Json } from '../types/json'
+
+type RpcRequest = {
+  method: string
+  params: Json[]
+  id: number
+}
+
+function createKey(url: string, requestBody: Json) {
+  return createHash('md5')
+    .update(url + JSON.stringify(requestBody))
+    .digest('hex')
+}
```

Comment:
Added Url to the hash just in case two different chains have the same parameters on a request.